.storybook/components/DesignTokens/Tier3/TypographyComponents.stories.tsx

```diff
@@ -3,12 +3,13 @@ import { at, capitalize, forEach, merge } from 'lodash';
 
 import React from 'react';
 import { Grid, Section } from '../../../../src';
-import breadcrumb from '../../../../src/design-tokens/tier-3-component/breadcrumb.json';
-import button from '../../../../src/design-tokens/tier-3-component/buttons.json';
-import form from '../../../../src/design-tokens/tier-3-component/forms.json';
-import link from '../../../../src/design-tokens/tier-3-component/link.json';
-import tab from '../../../../src/design-tokens/tier-3-component/tab.json';
-import tag from '../../../../src/design-tokens/tier-3-component/tag.json';
+import breadcrumb from '../../../../src/design-tokens/tier-3-component/breadcrumb-typography.json';
```

Comment:
breaking apart the per-component token files to the types of content. style-dictionary rolls up all the token files together so there's nothing relying on the input files to have all the contents like before.

Will continue this pattern, as there can be different pipelines for token generation based on the token content (in this case, colors and references).